internal/commands/parse.go

```diff
@@ -12,12 +12,30 @@ import (
 	"github.com/spf13/viper"
 )
 
+const parseDesc = `
+This command prints the internal representation of your input files.
+
+This can be useful in helping to write Rego policies. It is not always clear how 
+your input file will be represented in the Rego policies. The type of the input is inferred
+based on the file extension. If inference is not possible (e.g. due to the file coming from stdin)
+the '--input' flag can be used to explicitly set the input type, e.g.:
+
+	$ conftest parse --input toml
+
+See the documentation of the '--input' flag for the supported input types.
+
+The policy location defaults to the policy directory in the local folder.
```

Comment:
`--policy` is not required for `parse` command. (and not available)
It's enough to take inputs as a list